docs/architecture/concepts/re_scheduler.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
```

Comment:
This is a proof of concept code which I thought would be a candidate for a simpler cont scheduler.  It is not fit for using, yet, but I left it as a place holder for future work.